core/src/main/java/apoc/merge/Merge.java

```diff
@@ -46,7 +75,7 @@
         String identPropsString = buildIdentPropsString(identProps);
 
         final String cypher = "MERGE (n:" + labels + "{" + identPropsString + "}) ON CREATE SET n += $onCreateProps ON MATCH SET n += $onMatchProps RETURN n";
-        return tx.execute(cypher, params ).columnAs("n").stream().map(node -> new NodeResult((Node) node));
+        return tx.execute(cypher, params);
```

Comment:
magari ci puoi aggiungere:
```suggestion
        return tx.execute(cypher, params).columnAs("n");
```

Reply:
Per com'è fatto, ho bisogno di ritornare `Result` dal metodo, perché poi lo utilizzo per fare `nodeResult.getQueryStatistics()` alla riga 64

Reply:
Ok